src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java

```diff
@@ -2142,6 +2321,11 @@ public MemberCacheViewImpl getMembersView()
         return memberCache;
     }
 
+    public SortedSnowflakeCacheViewImpl<AutoModerationRule> getAutoModerationRulesView()
+    {
+        return autoModerationRuleCache;
+    }
```

Comment:
You already created `getAutoModerationRuleCache`?

Reply:
I need it for AutoModerationRuleDeleteHandler

Reply:
Why? AUTO_MODERATION_RULE_DELETE includes the rule that was deleted as the payload. I don't see a reason to cache these at all.

Reply:
oh ok